app/forms/reactivate_profile_form.rb

```diff
@@ -9,9 +9,11 @@ class ReactivateProfileForm
   attr_accessor :recovery_code, :password
   attr_reader :user
 
-  def initialize(user, attrs = {})
+  def initialize(user, attrs = { recovery_code: [] })
```

Comment:
I get that, but we do want a fallback in the case of recovery_code being `nil`, correct? In the `show` action we call `@reactivate_profile_form = ReactivateProfileForm.new(current_user)`, which obviously doesn't include a recovery code.

Is it better practice to supply an explicit argument vs rely on a default param in the function signature?